docusaurus.config.js

```diff
@@ -206,6 +213,211 @@ const config = {
         anonymizeIP: true,
       },
     ],
+    [
+      "@docusaurus/plugin-client-redirects",
```

Comment:
These are redirects from old vuepress links. We need the same for all links from the current docusaurus (for all versions given that links have changed in all versions) to the new links.